gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/explore.proto

```diff
@@ -47,6 +48,7 @@ message EntityOption {
 
 message ExploreResponse {
   repeated org.hypertrace.gateway.service.v1.common.Row row = 2;
+  int32 total = 3;
```

Comment:
Don't think, we would ever cross the `int32` limits, but, it's better to have it `int64` just in case we have an entity that has so many rows (perhaps in the future).